README.md

```diff
@@ -2,12 +2,9 @@
 
 [![CircleCI](https://circleci.com/gh/spotify/bigtable-autoscaler.svg?style=svg)](https://circleci.com/gh/spotify/bigtable-autoscaler)
 
-You have a Bigtable cluster and you would like to optimize its cost by using the
-right number of nodes at any given time. Then you should consider using the Bigtable
-autoscaler service!
-
-Many Bigtable clusters have uneven load over time. To avoid wasting capacity (and money), it's 
-desirable to scale down the cluster during off-hours. The Bigtable autoscaler lets you do that 
+If you have a Bigtable cluster and you would like to optimize its cost by using the
```

Comment:
Since we are changing a bit this text, can we say
"If you have a Bigtable cluster and you would like to optimize its **cost-efficiency** by using ..." 
(small suggestion, you don't need to accept)

Reply:
Great suggestion, fixed!